brave/browser-laptop
# Add option to disable top site suggestions

Closes #4977

Auditors:

Test Plan 1:
1. Open about:preferences#search
2. Disable `Show top site matches`
3. Open a new tab
4. Input `face`
5. Make sure that there does not appear the suggestions list

Test Plan 2:
1. Open about:preferences#search
2. Enable `Show top site matches`
3. Open a new tab
4. Input `face`
5. Make sure there appears the suggestions list, which has facebook.com

Submitter Checklist:



Reviewer Checklist:

Tests
